<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataviewForm</class>
 <widget class="QWidget" name="dataviewForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data View</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="topMargin">
      <number>16</number>
     </property>
     <property name="bottomMargin">
      <number>16</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>-1</number>
       </property>
       <property name="rightMargin">
        <number>24</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label1">
           <property name="text">
            <string>START DATE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="startDate">
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label2">
           <property name="text">
            <string>STOP DATE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="stopDate">
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="label3">
           <property name="text">
            <string>Dev ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbDevid">
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label4">
           <property name="text">
            <string>Field Type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbField">
           <item>
            <property name="text">
             <string>Flow</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Total</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Percent</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnQuery">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>13</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>14</number>
         </property>
         <property name="bottomMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>471</height>
            </size>
           </property>
           <column>
            <property name="text">
             <string>Date</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Time</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Devid</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Values</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
